proxy/src/main/java/org/apache/accumulo/proxy/Proxy.java

```diff
@@ -135,7 +136,8 @@ public static TServer createProxyServer(Class<?> api, Class<?> implementor, fina
     @SuppressWarnings("unchecked")
```

Comment:
Before, this suppression was fine to do here, but I think the cast to `Class<TProcessor>` should be moved up to the assignment of `proxyProcClass` instead. That way, the cast here for the `Constructor` is unnecessary, and makes it easier to follow to the next line when the `Constructor` is instantiated with the `RpcWrapper`.